src/actions/channels.js

```diff
@@ -889,6 +889,37 @@ export function autocompleteChannels(teamId, term) {
     };
 }
 
+export function autocompleteChannelsForSearch(teamId, term) {
+    return async (dispatch, getState) => {
+        dispatch({type: ChannelTypes.GET_CHANNELS_REQUEST}, getState);
+
+        let channels;
+        try {
+            channels = await Client4.autocompleteChannelsForSearch(teamId, term);
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch, getState);
+            dispatch(batchActions([
+                {type: ChannelTypes.GET_CHANNELS_FAILURE, error},
+                logError(error)(dispatch),
```

Comment:
this should be `logError(error)` please remove `(dispatch)`

Reply:
Good point. Now fixed.